src/components/application_manager/src/help_prompt_manager_impl.cc

```diff
@@ -329,20 +327,18 @@ void HelpPromptManagerImpl::CreateVRMsg(
       out_msg_params[strings::vr_help_title] = app_.name();
     }
   }
+
   out_msg_params[strings::vr_help] =
       smart_objects::SmartObject(smart_objects::SmartType_Array);
-  uint32_t index = 0;
-  for (size_t i = 0; i < vr_commands_.size(); ++i) {
+  const size_t count_to_add = std::min(vr_commands_.size(), kLimitCommand);
```

Comment:
Same comment as above.

Reply:
@mrapitis VR items generation was extracted to a separate function. See https://github.com/smartdevicelink/sdl_core/pull/2557/commits/c70ce73f88a4f45112f4f1044f77d4c331cdd2f2